projects/plugins/jetpack/_inc/client/recommendations/prompts/site-type/index.jsx

```diff
@@ -106,6 +109,7 @@ const SiteTypeQuestionComponent = props => {
 				'jetpack'
 			) }
 			answer={ answerSection }
+			isLoadingSideContent={ isFetchingSuggestions }
```

Comment:
Background is decided in `PromptLayout`, I believe we still need to pass the prop here if that was your point.

Reply:
I was thinking that since `<DiscountCard />` already has a check for `isLoading`:

```
const isLoading = isFetchingDiscount || isFetchingSuggestions || isFetchingOffers;
```

That the `isLoading` value in `<DiscountCard />` could be used within the `render` method of `<DiscountCard />` to show some alternate output if the data needed for the card is not yet available. Then, there would not be as much need for this `isLoadingSideContent` prop, which does similar data availability checks - just higher up in the component tree.

Reply:
The card container does need that information too, though. But maybe less so one the illustrations have been updated. Let's revisit that later.